traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/dns/ZoneManager.java

```diff
@@ -820,8 +934,7 @@ private static Record createRecord(final Name name, final InetRecord address) th
 		final List<Record> records = new ArrayList<Record>();
 		records.add(staticZone.getSOA());
 
-		@SuppressWarnings("unchecked")
-		final Iterator<Record> ns = staticZone.getNS().rrs();
+		@SuppressWarnings("unchecked") final Iterator<Record> ns = staticZone.getNS().rrs();
```

Comment:
I kind of like these annotations better the way they were

Reply:
done